src/Estimators/EstimatorManagerCrowd.cpp

```diff
@@ -31,32 +31,35 @@ EstimatorManagerCrowd::EstimatorManagerCrowd(EstimatorManagerNew& em)
   for (int i = 0; i < em.Estimators.size(); i++)
     scalar_estimators_.push_back(em.Estimators[i]->clone());
   MainEstimator = scalar_estimators_[EstimatorMap[MainEstimatorName]];
-  for(UPtr<OperatorEstBase>& upeb : em.operator_ests_)
+  for (UPtr<OperatorEstBase>& upeb : em.operator_ests_)
   {
     operator_ests_.emplace_back(upeb->clone());
   }
 }
 
 void EstimatorManagerCrowd::accumulate(int global_walkers, RefVector<MCPWalker>& walkers, RefVector<ParticleSet>& psets)
-  {
-    block_weight_ += walkers.size();
-    //Don't normalize we only divide once after reduction.
-    //RealType norm             = 1.0 / global_walkers;
-    int num_scalar_estimators = scalar_estimators_.size();
-    for (int i = 0; i < num_scalar_estimators; ++i)
-      scalar_estimators_[i]->accumulate(global_walkers, walkers, 1);
-    for (int i = 0; i < operator_ests_.size(); ++i)
-      operator_ests_[i]->accumulate(walkers, psets);
-  }
+{
+  block_weight_ += walkers.size();
+  //Don't normalize we only divide once after reduction.
+  //RealType norm             = 1.0 / global_walkers;
+  int num_scalar_estimators = scalar_estimators_.size();
+  for (int i = 0; i < num_scalar_estimators; ++i)
+    scalar_estimators_[i]->accumulate(global_walkers, walkers, 1);
+  for (int i = 0; i < operator_ests_.size(); ++i)
+    operator_ests_[i]->accumulate(walkers, psets);
+}
 
 
 void EstimatorManagerCrowd::startBlock(int steps)
 {
   crowd_estimator_timer_.restart();
+  for (auto& uope : operator_ests_)
+  {
```

Comment:
More of my taste it is not worth adding {} around a single line.

Reply:
I agree with you on that sometimes I just drop the braces and forget to clean up.